doc/man/8/rbd.rst

```diff
@@ -361,6 +361,11 @@ Commands
 :command:`bench-write` [--io-size *size-in-B/K/M/G/T*] [--io-threads *num-ios-in-flight*] [--io-total *total-size-to-write-in-B/K/M/G/T*] [--io-pattern seq | rand] *image-spec*
```

Comment:
Do you mean remove the bench-write related entry from manpage? Why not to keep it with a Deprecated warning until we remove it actually from rbd command?